src/components/NavigationBar.js

```diff
@@ -5,14 +5,15 @@ import {
   Route,
   Link,
   Redirect,
-} from 'react-router-dom';
-import { UsersPanel } from 'components/Users/UsersPanel';
-import { MountsPanel } from 'components/Mounts/MountsPanel';
-import { UserMountsPanel } from 'components/UserMounts/UserMountsPanel';
-import { getMountsService } from 'services/mountServices';
-import { getUsersService } from 'services/userServices';
-import { getUserMountsService } from 'services/userMountsServices';
-import { Tabs, Tab, AppBar, Paper } from '@material-ui/core/';
+} from "react-router-dom";
+import { UsersPanel } from "./Users/UsersPanel";
+import { MountsPanel } from "./Mounts/MountsPanel";
+import { UserMountsPanel } from "./UserMounts/UserMountsPanel";
+import { HelpPanel } from "./Help/HelpPanel";
```

Comment:
Bad, out of date formatting. HelpPanel import is needed. this has been corrected.

Reply:
Revert these imports to absolute imports.

Reply:
all relative imports in NavigationBar.js returned to absolute imports